spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -118,3 +118,40 @@ A full list of changes may be found on the link:https://github.com/eclipse/micro
 
 - Time values in TCK tests are now configurable (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/399[#399]).
 - Transitive dependency on `javax.el-api` has been excluded (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/439[#439]).
+
+[[release_notes_30]]
+== Release Notes for MicroProfile Fault Tolerance 3.0
+
+This release is a major release of Fault Tolerance which includes backward incompatible changes.
+
+A full list of changes can be found on the link:https://github.com/eclipse/microprofile-fault-tolerance/milestone/9[MicroProfile Fault Tolerance 3.0 Milestone]
+
+=== Backward incompatible changes
+
+==== Metric names and scopes changed
+
+The metrics added automatically by MicroProfile Fault Tolerance have been updated to take advantage of support for metric tags which was added to MicroProfile Metrics in version 2.0. As a result, some information which was previously contained in the metric name is now instead included in tags.
+
+In addition, metrics have moved from the `application:` scope to the `base:` scope for consistency with other MicroProfile specifications. Note that this means:
+
+* metrics are exported under `/metrics` and `/metrics/base`, instead of `/metrics` and `/metrics/application`
```

Comment:
Metrics are now exported under `/metrics` and `/metrics/base`, instead of `/metrics` and `/metrics/application` in the previous versions.